libs/components/flyout/src/lib/modules/flyout/flyout.service.spec.ts

```diff
@@ -36,7 +38,10 @@ describe('Flyout service', () => {
   ));
 
   it('should only create a single host component', () => {
-    const spy = spyOn(service as any, 'createHostComponent').and.callThrough();
+    const spy = spyOn(
```

Comment:
Can you not mock out `SkyDynamicComponentService` with a Jasmine SpyObj? If not, you can use `TestBed.inject()` to get the instance of `SkyDynamicComponentService` and then spy on its `createComponent` method.